config/coverband.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# config/coverband.rb NOT in the initializers
+Coverband.configure do |config|
+  config.store = Coverband::Adapters::RedisStore.new(Redis.new(url: Settings.redis.app_data.secondary_url))
```

Comment:
```suggestion
  config.store = Coverband::Adapters::RedisStore.new($redis)
```

I don't know if the secondary url will even be set here. App data is the right spot but $redis will already be initialized with a Redis instance.